src/web/collectors/CollectorsList.jsx

```diff
@@ -70,40 +81,54 @@ const CollectorList = createReactClass({
     const { sort } = this.state;
     const field1 = sort(collector1);
     const field2 = sort(collector2);
+
     return (this.state.sortDesc ? naturalSort(field2, field1) : naturalSort(field1, field2));
   },
 
-  _getTableHeaderClassName(field) {
-    return (this.state.sortBy === field ? (this.state.sortDesc ? 'sort-desc' : 'sort-asc') : 'sortable');
+  _getSortingIcon(field) {
+    const { sortBy, sortDesc } = this.state;
+    const name = (sortBy === field ? (sortDesc ? 'sort-amount-up' : 'sort-amount-down') : 'sort');
+
+    return <StyledIcon name={name} fixedWidth $isAlwaysVisible={sortBy === field} />;
   },
 
   _formatCollectorList(collectors) {
     return (
       <div className="table-responsive">
-        <table className="table table-striped collectors-list">
+        <Table className="table table-striped collectors-list">
```

Comment:
Sorry I missed this on the first round, but I believe this can be simplified to:

```suggestion
        <Table striped>
```

Not sure what the className `collectors-list` is referencing, but you can add it back in if it's necessary.

Reply:
Good point, forgot about that. The `collectors-list` class is setting some margins, and so are another couple classes defined in the common stylesheet file that I missed before. I'll require the file again, since this is deprecated code and that way is safer and faster to do.

Reply:
In the end I added the margins in a styled-component. It seems other classes for this file either had wrong selectors or were not really needed.